tests/requirements.txt

```diff
@@ -4,4 +4,4 @@ pylint==2.15.5  # Guard against warn/error changes
 jsonschema==4.7.2
 tree-sitter==0.20.1  # For parsing C++ AST to assert on comment text.
 tree-sitter-languages==1.5.0
-#openassetio>=1.0.0b1
+openassetio>=1.0.0b1
```

Comment:
[Question] Are we happy with Python and C++ diverging once a subsequent beta is released? if I wonder if we should pre-install `v1.0.0b1rev0` and it will be happy with that ?

Reply:
Pre-install? Do you mean pin to b1.rev0?

But good point in general.  I've seen the strategy for semver releases in GitHub where you get tags for each semver "prefix", so that when you release e.g. 1.0.1 then the new v1.0.1 tag is created, and so bumps the v1 tag and the v1.1 tag.

If we did that, and it worked for beta releases in a similar way, then we could pin the C++ checkout to `v1.0.0.beta`, which would match any beta release.  Not quite the same as what we're saying here for Python, so perhaps we could add `<v1.0.1` to make them line up.

Though this seems like a lot of work to do and test at the moment, and might just not make sense for pre-releases.

So I think your suggestion of pinning to rev0 is the best solution for now.

Reply:
Took the liberty of force-pushing (7ebba945)

Reply:
Oh sorry I forgot this was a test requirement vs a runtime one.

I guess I Was thinking that if we pin it here then when we install it in integrations, it'll get upset once there is a new release of `openassetio`? So we could use a greater than requirement here (so integrations still work), and have the CI step manually install the same exact version a the C++ tests (until we do something like you say above), which should still satisfy this requirement.

Reply:
Got you, good point. Updated in caab3f61